packages/adp-tooling/src/base/helper.ts

```diff
@@ -13,6 +17,57 @@ export function getVariant(basePath: string): DescriptorVariant {
     return JSON.parse(readFileSync(join(basePath, 'webapp', 'manifest.appdescr_variant'), 'utf-8'));
 }
 
+/**
+ * Writes the updated variant content to the manifest.appdescr_variant file.
+ *
+ * @param {string} basePath - The base path of the project.
+ * @param {DescriptorVariant} variant - The descriptor variant object.
+ * @param {Editor} fs - The mem-fs editor instance.
+ */
+export function updateVariant(basePath: string, variant: DescriptorVariant, fs: Editor) {
+    fs.writeJSON(join(basePath, 'webapp', 'manifest.appdescr_variant'), variant);
+}
+
+/**
+ * Determines whether the project at the given base path is an Adaptation Project (ADP).
+ *
+ * @param {string} basePath - The base path to the project.
+ * @returns {Promise<boolean>} A promise that resolves to true if the project is an ADP project, or false otherwise.
+ * @throws {Error} If the project type cannot be determined.
+ */
+export async function isAdpProject(basePath: string): Promise<boolean> {
```

Comment:
Can we use https://github.com/SAP/open-ux-tools/blob/c59e479be60306b5e7807ba7f6ca46f45fad8c62/packages/project-access/src/project/info.ts#L139 instead?

Reply:
Changed in: [c295d36](https://github.com/SAP/open-ux-tools/pull/2610/commits/c295d3650878c552b5c692db7d878e7d02e84341)